### PR TITLE
crud-server, test: create test user via dotEnv file

### LINKED ISSUE #140: move test user to `.env` file
currently the user that is used for the tests is hard-coded in the test suite. we have to move the test suite to use an arbitrary user that the tester can define. we already use `.env.conduit` for storing conduit information related to the external service. we could move the user details into the `.env.conduit` file so that the user can be defined on-demand.

the advantage is that this same user can be re-used to login to test the front-end. the test conduits that is created by the backend test suite can also give a head-start for the tester testing the front-end.

### PR DESCRIPTION
**Pull request by shinenelson**
#### TL;DR version
- move `dotEnv` configuration to `model.test.js`
- rename `.env.conduit` to `.env.conduit-user`
- move creation of test conduits for proxy server to `model.test.js`
- create new user from dotEnv
- use the newly created user for the test suite
- use a base object for common conduit parameters for proxy server
- add 200 dummy conduits to user for easier front-end testing

closes #140

note : this changeset might create a merge conflict with #139 since both
pulls touch `crud-server/src/routes.test.js`. specifically the setting 
of the `dotEnvValues`.

---

#### longer prose-style version
the tests were running with dummy users that were generated dynamically
during the invocation of the tests. while this is ok in the scope of the
tests alone, it makes more sense to use a user that is specified by the
tester via a `.env` file. this also enables easy testing of the front-end
application as well since this avoids the 2 pre-requisite steps for
front-end testing - create a user ( after each database flush ), and
create multiple conduits. this manual process is also time consuming,
hence warranting for an automated process to set this up so that the
tester can get started sooner.

move creation of test conduits for the proxy server from `routes.test.js`
to `model.test.js` since there are more conduits being generated for the
user and also because it is easier to write directly into the database
than make a REST API request for the creation of multiple conduits.

### CODE REVIEW

**Comment by m5nv** [comment]
LGTM. Thanks.
